etc/env/base/contracts.toml

```diff
@@ -41,7 +41,7 @@ L2_WETH_TOKEN_PROXY_ADDR="0x5E6D086F5eC079ADFF4FB3774CDf3e8D6a34F7E9"
 FRI_RECURSION_LEAF_LEVEL_VK_HASH ="0x062362cb3eaf1f631406cbe19bf2a2c5d0d9ea69d069309a6003addae9f387be"
 FRI_RECURSION_NODE_LEVEL_VK_HASH ="0x5a3ef282b21e12fe1f4438e5bb158fc5060b160559c5158c6389d62d9fe3d080"
 FRI_RECURSION_SCHEDULER_LEVEL_VK_HASH ="0xf12256f7ba1ab223ba82abb19c8366c6677596897f9e5a82016977113e52ed5d"
-SNARK_WRAPPER_VK_HASH = "0x8574e152c41dc39a2ecab984545e1cf21cb3ec250b919018a8053f2fa270784f"
+SNARK_WRAPPER_VK_HASH = "0x750d8e21be7555a6841472a5cacd24c75a7ceb34261aea61e72bb7423a7d30fc"
```

Comment:
revert

Reply:
(btw - where did it come from?)

Reply:
it came from the contracts via genesis init

Reply:
i looked more into it, this is the value that was used before the new fee model changes. i'm not sure why the version of the contracts repo is computing the old value